spark/src/main/kotlin/com/adevinta/spark/components/divider/Divider.kt

```diff
@@ -67,17 +67,16 @@ public enum class LabelVerticalAlignment {
     Bottom,
 }
 
-@Deprecated(
-    "Divider is deprecated",
-    ReplaceWith("HorizontalDivider()"),
-)
 /**
  * A deprecated Divider Component.
  *
  * @param modifier The modifier to be applied to the divider.
  * @param color The color of the divider.
  */
-
+@Deprecated(
+    "Divider is deprecated",
+    ReplaceWith("HorizontalDivider()"),
+)
```

Comment:
Why would we add modifier = modifier?

Reply:
We need the modifier argument otherwise the one used previously will be lost.
You can read more info online on this like [this article](https://dev.to/mreichelt/the-hidden-kotlin-gem-you-didn-t-think-you-ll-love-deprecations-with-replacewith-3blo) or the [ReplaceWith documentation](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-replace-with/expression.html)